src/hooks/useNotificationTypes.tsx

```diff
@@ -210,7 +210,7 @@ export const notificationTypes: NotificationTypeConfig[] = [
       useEffect(() => {
         // eslint-disable-next-line no-restricted-syntax
         for (const transfer of transferNotifications) {
-          const { fromChainId, status, txHash, toAmount, type, isExchange } = transfer;
+          const { id, fromChainId, status, txHash, toAmount, type, isExchange } = transfer;
```

Comment:
[q] do transferNotifications have an `id`? i thought they have a `txHash` or `requestId`

Reply:
^ just saw that you added an id property!